common/src/main/java/org/geysermc/floodgate/AbstractFloodgateAPI.java

```diff
@@ -17,17 +17,24 @@ public static FloodgatePlayer getPlayer(UUID uuid) {
     }
 
     /**
-     * Method to determine if the given <b>online</b> player is a bedrock player
-     * @param uuid The uuid of the <b>online</b> player
-     * @return true if the given <b>online</b> player is a Bedrock player
+     * Method to determine if the given online player is a Bedrock player
+     * @param uuid The uuid of the player
+     * @return true if the given player is a Bedrock player
      */
     public static boolean isBedrockPlayer(UUID uuid) {
+        // return uuid.version() == 3;
         return players.containsKey(uuid);
     }
 
-    /**
-     * Create a valid Java player uuid of a xuid
-     */
+//    /**
+//     * Method to determine if the given player is a Java player
+//     * @param uuid The uuid of the player
+//     * @return true if the given player is a Bedrock player
+//     */
+//    public static boolean isJavaPlayer(UUID uuid) {
+//        return uuid.version() == 4;
+//    }
```

Comment:
Should this not just be removed?

Reply:
No, this will be used later again.

Reply:
Oh, I removed it

Reply:
Added back in https://github.com/GeyserMC/Floodgate/pull/18/commits/1ee2e31b2d35aa7d761acad623c11ec059b42e26